repo-metadata.html

```diff
@@ -2,18 +2,7 @@
 title: repo metadata
 ---
 
-﻿<!doctype html>
-<html>
-{% include head.html %}
-<body>
-    {% include header.html %}
-    <div class="wrapper">
-        <section>
-            {% include title.html %}
-            <strong>site.github</strong>
-            {{ site.github }}
-        </section>
-        {% include footer.html %}
-    </div>
-</body>
-</html>
+{% for x in site.github %}
```

Comment:
You could do `{{ site.github | jsonify }}` instead here to get the whole hierarchy